### DISCUSSION TITLE
Where the information about the most used applications in drun mode is stored

### DISCUSSION THREAD

**Discussion by istudyatuni** [davatorium/rofi#1613] [Q&A] [answered]
Now I use wofi (rofi for wayland), but rofi has the same feature - show the most used applications at the top. I tried to find answer to this question on google and duckduckgo, but there is only irrelevant information (like "Top 10 most used Linux applications"). My goal is to reset or change this information

Edit (after locking): anyway, thank you, I found it in `~/.cache`. I asked it here because I couldn't find any discussion of wofi, sorry about that

**Comment by DaveDavenport** [comment, marked as answer]
This is not the wofi support forum and we have 0 links to wofi. I have no idea what wofi does. 
wofi is not a port of rofi, see nxt message for an actual port (based on same code bas).

ROFI stores it in: XDG_CACHE_HOME  or the cache dir the user specified in rofi config file.

**Reply by DaveDavenport** [reply]
If you use https://github.com/lbonn/rofi we might be able to help (wayland port of rofi.).

**Comment by DaveDavenport** [comment]
locking as this does not belong here.
